### PR TITLE
[logging] Add subscription tests

### PR DESCRIPTION
**Pull request by mgirgisf**

### CODE REVIEW

**Comment by mgirgisf** [comment]
recheck

**Review by vyzigold — APPROVED** [review]

**Comment by mgirgisf** [comment]
recheck

**Review by ayefimov-1 — APPROVED** [review]

**Comment by mgirgisf** [comment]
recheck

File: roles/common/README.md [hunk 1]
```diff
@@ -82,6 +91,16 @@ can be set at the play level.
...
           name: common
 
+
+      - name: "Verify subscription"
+        ansible.builtin.import_role:
+          name: common
+        vars:
+          common_subscription_test_id: "RHOSO-12678"
+          common_subscription_nspace: openshift-operators-redhat
+          common_subscription_l
```
Comment (elfiesmelfie) [inline roles/common/README.md:101]: This needs to be updated

Comment (elfiesmelfie) [inline roles/common/README.md:101]: this needs to be updated as it is incomplete

**Review by elfiesmelfie — APPROVED** [review]

**Review by elfiesmelfie — APPROVED** [review]

**Pull request merged**